.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -235,14 +205,12 @@ steps:
       ${CONTAINER_IMAGE} make -f packer.mk generate-sas
     displayName: Getting Shared Access Signature URI
     condition: and(succeeded(), eq(variables.DRY_RUN, 'False'), 
-                   or(and(eq(variables['MODE'], 'sigMode'), eq(variables.GEN2_SIG_FOR_PRODUCTION, 'True')), 
-                      eq(variables['MODE'], 'default')))
+                   and(eq(variables['MODE'], 'windowsVhdMode'), eq(variables.SIG_FOR_PRODUCTION, 'True')))
```

Comment:
do we have only one mode after this PR is merged? If so, maybe we can remove this check `eq(variables['MODE'], 'windowsVhdMode')`